### PR TITLE
Bump version to v8.4.0.27

### PR DESCRIPTION
**Pull request by IvanYashchuk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by IvanYashchuk** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
@IvanYashchuk Sorry, could you merge #46 first? Thanks 😅

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,13 +13,13 @@ package:
...
+    sha256: 7ef72353331cf42b357f53cb4a4971fb07e2f0b2ae66e03d54933df52de411c8  # [ppc64le and cuda_compiler_version == "11.2"]
   - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-linux-sbsa-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.tar.xz  # [aarch64]
-    sha256: 7eb8c96bfeec98e8aa7cea1e95633d2a9481fc99040eb0311d31bf137a7aa6ea  # [aarch64 and cuda_compiler_version == "11.2"]
+    sha256: 3972ab37b6f0271274931f69c5675c3b61d16f8f5a2dedd422a5efd7b0f358e5  # [aarch64 and cuda_compiler_version == "11.2"]
   - url: https://developer.download.nvidia.com/compute/redist/cudnn/v{{ ".".join(version.split(".")[:3]) }}/local_installers/{{ cudnn_cuda_version }}/cudnn-windows-x86_64-{{ version }}_cuda{{ cudnn_cuda_version }}-archive.zip  # [win64]
-    sha256: 9e36eef803f1cf9ab24846dc133a3014fdc548775ee29073e8466d415957a1c0  # [win64 and cuda_compiler_version == "11.2"]
+    sha256: 9885484d4f3dc8c25c6674225751bd0af1c80fbb2a77227d1439b917ec16ec92  # [win64 and cuda_compiler_version == "11.2"]
 
 build:
   number: 0
```
Comment (leofang) [inline recipe/meta.yaml:25]: Need to bump this

**Review by leofang — CHANGES_REQUESTED** [review]

**Comment by ngam** [comment]
Thanks @IvanYashchuk, this is ready 💚

**Pull request merged**